.github/workflows/clippy.yml

```diff
@@ -15,21 +15,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-
-    - name: Install Nightly Toolchain
-      uses: actions-rs/toolchain@v1
-      with:
-        profile: minimal
-        toolchain: nightly
-        components: rustfmt
-        override: true
-
-    - name: Check formatting
-      run: |
-        cargo fmt --all -- --check
-
-    - name: Lint with Clippy
-      uses: actions-rs/clippy-check@v1.0.7
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
+      - uses: actions/checkout@v2
+      - run: rustup component add clippy rustfmt
+      - uses: actions-rs/clippy-check@v1
```

Comment:
Not, but maybe add names for the checkout and clippy steps? Just makes things a bit cleaner/easier to read I think.

Other than that, LGTM.